app/display/editor/doc/access_widget.rst

```diff
@@ -6,546 +6,25 @@ Access Widget in Script
 **The widget object**
 
 The widget to which a script is attached can be accessed in the script via **widget** object. It is the
-controller (or EditPart in GEF) of the widget. The widget object provides methods to get or set any of its properties,
+controller of the widget. The widget object provides methods to get or set any of its properties,
 store external objects or provide special methods for a particular widget.
 
-**The display object**
+**Access to widgets in a display**
 
-The widget controller of the display is accessible in all scripts as a **display** object. To get the controller of any
-widget in the display, you can call its method ``getWidget(name)``. For example:
+A script may obtain a list of all widgets in a display like so:
 
-.. code-block:: javascript
+.. code-block:: python
 
-    display.getWidget("myLabel").setPropertyValue("x", 20); //set x position of the widget myLabel to 20.
+    widget.getDisplayModel().getChildren()
 
-**Common methods to all widgets**
+To find a specific widget the script may then loop over the list and use `getName()` to match on the widget
```

Comment:
Looping over the child widgets will only consider the direct children, not widgets within a group or tab.
The suggested way to find widgets by name is `ScriptUtil.findWidgetByName`

Reply:
OK, I will update.

Reply:
Great, then go ahead and merge,